backend/core/dependencies.py

```
@@ -0,0 +1,8 @@
+from database.session import SessionLocal
+# Dependency to get DB session
+def get_db():
```

Ruff (0.8.2)

Missing return type annotation for public function `get_db`

(ANN201)